### ISSUE TITLE
MSID: 91469 Version: 1 DOI: 2023.08.30.23294826

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#1302]
[PLACE MANUSCRIPT AND EDITOR DETAILS HERE WHEN AVAILABLE] See step 3

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [ ] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v2/by-publisher/elife/get-by-manuscript-id?manuscript_id=91469)

or (only one should be ticked. remove other from description.)

- [ ] Confirmed with Ted at bioRxiv that MECA archive is available

Send the following email to Ted and wait for his reply.

```
Hi Ted,

Please can you prepare the preprint MECA for 10.1101/2023.08.30.23294826

Thanks
```

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/91469/v1)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: [PLACE LINK TO PULL REQUEST HERE]

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews and QC

Who can help: Production team

- [ ] QC passed or QC not needed

Who can help: Editorial team

- [ ] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2023.08.30.23294826)
- [ ] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [ ] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/91469/v1/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/91469)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/91469
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/91469
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/91469/v1/91469-v1.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/91469)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by rkcook312** [comment]
Missing corresponding email on eLife preprint

Poor resolution / readability for Figure 5 and Supplementary Figure S1

Double bullet points after the sentence "Additional analyses that may be performed during the efficacy phase include:" on eLife preprint

The list after the below sentence is formatted using numbers in medRxiv but bullet points on eLife preprint?
During the PAVE study, four AI algorithms are being developed and evaluated: (1) cervix detector, (2) image quality classifier, (3) disease classifier to identify precancer, and (4) treatability/SCJ classifier.

**Comment by fred-atherden** [comment]
Nicely spotted @rkcook312. Are you OK to reach out to the authors about the poor readability of figures 5 and S1? 

The correspondence issue we'll have to reassess once they post an update version. 

Double bullet points, the list indicator and the ref list are all platform issues that I think we can live with (although I accept they are not ideal and hope we can get these fixed soon).

**Comment by rkcook312** [comment]
Thanks @fred-atherden, I'll reach out to the authors to submit new version to medRxiv to fix the resolutions. Thanks for letting me know about the bullet points etc are platform issues, noted.

**Comment by rkcook312** [comment]
@fred-atherden The authors emailed to say they have submitted a new version to medRxiv (full text not appearing yet though). They also asked when can they submit the revised version with the reviewers comments? I guess they can do that once the RPP is published?

**Comment by fred-atherden** [comment]
Thanks Rebecca. We'll check in with this one in the next couple of days then. Hopefully medRxiv will process it soon. 

In terms of the revised version, I think they can now post it whenever they want now.

**Comment by rkcook312** [comment]
Great, thanks @fred-atherden 

The author has responded re: revised version. "Do I need to do the same process with Medrxiv? Can I say it has been accepted at eLife?"

I assume the answer is Yes to both. Just checking, don't want to give the authors incorrect information.

**Comment by fred-atherden** [comment]
Thanks Rebecca - I've no idea actually. Please could you forward the email to Editorial and ask for them to assist?

**Comment by fred-atherden** [comment]
The authors have posted a V2 with the issue (in theory) corrected - full text/XML not yet available. 

In the meantime @acollings are you OK to move the reviews over to the v2 version?

**Comment by acollings** [comment]
Done, @fred-atherden!

**Comment by fred-atherden** [comment]
Thanks!

**Comment by fred-atherden** [comment]
@rkcook312, the v2 has been loaded - could you check this one please?

**Comment by rkcook312** [comment]
@fred-atherden Mark Schiffman is missing an affiliation, this wasn't the case in v1? But image resolution and corresponding email issues noted before have been fixed.

**Comment by fred-atherden** [comment]
Thanks Rebecca. This is why:

```xml
<contrib contrib-type="author">
<contrib-id contrib-id-type="orcid">http://orcid.org/0000-0002-4625-2508</contrib-id>
<name><surname>Schiffman</surname><given-names>Mark</given-names></name>
</contrib>
<contrib contrib-type="author">
<name><surname>the PAVE Study Group</surname></name>
<xref ref-type="aff" rid="a1">1</xref>
</contrib>
```

bioRxiv have captured the affiliations incorrectly (and technically `the PAVE Study Group` should be an `<on-behalf-of>`)

**Comment by fred-atherden** [comment]
@rkcook312 should hopefully be fixed now - could you check?
